src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -128,7 +135,30 @@ export class OrganizationIntegrations extends AsyncComponent<
 
     const list = this.sortIntegrations(combined);
 
-    this.setState({list, displayedList: list}, () => this.trackPageViewed());
+    const allFeatureDescriptions: IntegrationFeature[] = list.reduce(
+      (acc, integration) => {
+        if (isSentryApp(integration)) {
+          const array = ['internal', 'unpublished'].includes(integration.status)
+            ? [{featureGate: integration.status, description: ''}]
+            : integration.featureData;
+          acc.push(...array);
+          return acc;
+        }
+        if (isPlugin(integration)) {
+          acc.push(...integration.featureDescriptions);
+          return acc;
+        }
+        acc.push(...integration.metadata.features);
+        return acc;
+      },
+      [] as IntegrationFeature[]
+    );
```

Comment:
nit: You could save some lines with just using `return acc.concat(integration.featureDescriptions)` for example.

Reply:
Looking at this again, is there any reason to use `.reduce` vs `.map`? I think every item in the list is being converted anyway, so the array isn't being reduced in size. 

If that's the case, I'd consider swapping over to `.map` instead ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map))

Reply:
@leeandher I don't think `.map` will work because it will return a nested array of `IntegrationFeatures[]`. Then I would have to loop over it again to flatten it.